amr-wind/utilities/tensor_ops.H

```diff
@@ -71,23 +71,27 @@ cross_prod(const T* a, const T* b, T* c)
 /** Compute the transformation `{y} = [A] {x}`
  */
 template <typename T>
-AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE void
-transform_vec(const T tmat[AMREX_SPACEDIM][AMREX_SPACEDIM], const T* x, T* y)
+AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE void transform_vec(
+    const amrex::Array2D<T, 0, AMREX_SPACEDIM, 0, AMREX_SPACEDIM> tmat,
+    const T* x,
+    T* y)
 {
-    y[0] = tmat[0][0] * x[0] + tmat[0][1] * x[1] + tmat[0][2] * x[2];
-    y[1] = tmat[1][0] * x[0] + tmat[1][1] * x[1] + tmat[1][2] * x[2];
-    y[2] = tmat[2][0] * x[0] + tmat[2][1] * x[1] + tmat[2][2] * x[2];
+    y[0] = tmat(0, 0) * x[0] + tmat(0, 1) * x[1] + tmat(0, 2) * x[2];
```

Comment:
Do we know that these are the same ordering?

Reply:
It defaults to Fortran ordering. Interestingly we don't ever use these functions. Nor do we test any of the functions in this file. I will add that to this PR.

Reply:
These functions are now tested in #1046. So I have the ordering right (fortran)